### ISSUE TITLE
Inclusion of dotenv creates undesirable side effects

### ISSUE THREAD

**Issue by grjones** [alexrudall/ruby-openai#123]
**Describe the bug**
As a library, it's undesirable to include new configuration methods. For us, including this in our Rails app made our ENV vars be set to values in `.env` instead of `.env.<environment>`. This is due to `ruby-openai` being loaded before Rails configuration is thereby loading `.env` first in a non-Rails context. Most anyone including this gem in Rails projects will run into this problem. It will be best to remove `dotenv` altogether.

Thank you for this gem!

**To Reproduce**
1. Create a Rails app
2. Have your Rails app use both `dotenv-rails` and `ruby-openai` in your Gemfile
3. Create `.env` and `.env.development` files with different values
4. Observe that values in `.env` are taking precedence over environment-specific values despite the ordering listed [here](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use).

**Expected behavior**
Environment-specific values should take precedence over `.env` per the dotenv README.  It's highly desirable to let users of this gem decide the methods they want to use to set configuration.

**Comment by alexrudall** [comment]
Hi @grjones thanks for this well defined issue, I'll look into it ASAP.

**Comment by grjones** [comment]
❤️

**Comment by jtoy** [comment]
Good catch!

**Comment by MatheusRich** [comment]
I ran into the same issue using this in a plain Ruby app. I was able to fix it using `Dotenv.overload!(env_file)`.

**Comment by grjones** [comment]
> I ran into the same issue using this in a plain Ruby app. I was able to fix it using `Dotenv.overload!(env_file)` in my app.

Yeah, that will work but still highly recommend we leave configuration methods up to the users of this lib. The README could be updated on installing dotenv and its usage, or at least having dotenv be off by default will prevent a lot of accidental misconfigurations when this lib is added.

**Comment by alexrudall** [comment]
Thanks all, hope to have v3 out within a week which will remove dotenv entirely ⚡️👍

**Comment by alexrudall** [comment]
Fix released in [v3.0.0](https://rubygems.org/gems/ruby-openai/versions/3.0.0) - thanks for raising this @grjones & others!

```ruby
    Ruby::OpenAI.configure do |config|
        config.access_token = ENV.fetch('OPENAI_ACCESS_TOKEN')
        config.organization_id = ENV.fetch('OPENAI_ORGANIZATION_ID') # Optional.
    end
```

**Comment by kilimchoi** [comment]
Hi @alexrudall I'm getting undefined method `configure' for Ruby::OpenAI:Module when I copy paste `Ruby::OpenAI.configure do |config|`, is this a known issue?
